README.md

```diff
@@ -528,6 +513,16 @@ Here are the steps:
 
         <img src=".//media/image35.png" style="width:5.38735in;height:3.18982in" alt="A screenshot of a cell phone Description automatically generated" />
 
+### Further instructions for IoT Plug and Play
+
+- For the IoT Plug and Play sample, download the IoT Plug and Play DTMI [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json).
```

Comment:
Will this be obsolete once microchip submits their changes?

Reply:
The link yes. The general instructions no. We can update it once they submit theirs